Common/src/geometry_structure.cpp

```diff
@@ -2695,8 +2705,17 @@ void CGeometry::RegisterOutput_Coordinates(CConfig *config){
   unsigned long iPoint;
 
   for (iPoint = 0; iPoint < nPoint; iPoint++){
-    for (iDim = 0; iDim < nDim; iDim++){
-      AD::RegisterOutput(node[iPoint]->GetCoord()[iDim]);
+    if(config->GetMultiphysicsDiscrete_Adjoint()) {
+      for (iDim = 0; iDim < nDim; iDim++) {
+//        AD::RegisterOutput(node[iPoint]->GetCoord()[iDim]);
```

Comment:
again if possible don't leave code commented out

Reply:
Ok. One would have to add those lines to set mesh adjoints. I thought this will be needed for FSI integration at some point but things seem to work out a little different now.
Should I add a comment and we will then uncomment or delete it in the course of integration of FSI into this driver?

Reply:
Yup leave a comment then